mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+57d3883
```

Comment:
this file shouldn't be in a commit. see https://musescore.org/en/developers-handbook/compilation/compile-instructions-windows-mingw-git#Source-Code, the part about post-checkout hook